test/extended/cmd.sh

```diff
@@ -143,3 +144,15 @@ os::cmd::expect_success "oc process -f test/extended/fixtures/image-pull-secrets
 os::cmd::try_until_text "oc get pods/my-dc-1-hook-pre -o jsonpath='{.status.containerStatuses[0].imageID}'" "docker"
 os::cmd::expect_success "oc delete all --all"
 os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
+
+# check to make sure that service serving cert signing works correctly
```

Comment:
That's fine.

On Mon, May 2, 2016 at 1:36 PM, David Eads <notifications@github.com> wrote:

> In test/extended/cmd.sh
> <https://github.com/openshift/origin/pull/7728#discussion_r61772355>:
>
> > @@ -143,3 +144,15 @@ os::cmd::expect_success "oc process -f test/extended/fixtures/image-pull-secrets
> >  os::cmd::try_until_text "oc get pods/my-dc-1-hook-pre -o jsonpath='{.status.containerStatuses[0].imageID}'" "docker"
> >  os::cmd::expect_success "oc delete all --all"
> >  os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
> > +
> > +# check to make sure that service serving cert signing works correctly
>
> test/extended/cmd is not in conformance and therefore not tested on a
> normal PR job
>
> Wow, so we really do have bash scripts written in Go to call go code. It's
> like the worst of all possible worlds.
>
> @smarterclayton <https://github.com/smarterclayton> can I get a reprieve
> until we inject the CA so I can use podB to validate podA. Otherwise I'm
> hunting around trying to find bits of master config inside the test to find
> the CA certs in question and building an http handler by hand, all of which
> is a simple one-liner in bash.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7728/files/e3f4db09aa1f5cc71b10ecb2b076070ceedfde3f#r61772355>
>